git/repo.go

```diff
@@ -206,13 +206,13 @@ func (c *Checkout) TagRevision(tag string) (string, error) {
 func (c *Checkout) CommitsBetween(ref1, ref2 string) ([]Commit, error) {
 	c.RLock()
 	defer c.RUnlock()
-	return onelinelog(c.Dir, ref1+".."+ref2)
+	return onelinelog(c.Dir, ref1+".."+ref2, c.repo.GitRemoteConfig.Path)
```

Comment:
This expression and the one below don't need `.GitRemoteConfig`, because the GitRemoteConfig struct is embedded, so its fields are accessible directly.

(There may still be a good reason to spell out the path, in which case just comment here :)

Reply:
Forgot about this. Will change.